Core/RoverControllerPresets/MqttClasses.RoverControlVec3Extension.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Godot;
+using static RoverControlApp.Core.MqttClasses;
+
+namespace RoverControlApp.Core.RoverControllerPresets;
+
+public static class RoverControlVec3Extension
+{
+	public static Vector3 ToVector3(this RoverControl roverControl) =>
+		new((float)roverControl.Vel, (float)roverControl.XAxis, (float)roverControl.YAxis);
+
+	public static RoverControl FromVector3(Vector3 vector) =>
```

Comment:
Missing `this` keyword - thus it's no longer an extension...
should be `public static RoverControl FromVector3(this Vector3 vector)`. Also I'm not sure i like the wording:
```suggestion
	public static RoverControl ToRoverControl(this Vector3 vector) =>
```

Reply:
Adressed in [f98c87c](https://github.com/wisniax/RoverControlApp/pull/36/commits/f98c87cc0ebf98bac51edf63127e860968a00e50)